brainkeeper/src/app/components/session_screen/person-name/person-name.component.ts

```diff
@@ -1,15 +1,20 @@
 import { Component, OnInit } from '@angular/core';
 
+import { SessionService } from 'src/app/services/session-service';
+
 @Component({
   selector: 'app-person-name',
   templateUrl: './person-name.component.html',
   styleUrls: ['./person-name.component.scss']
 })
 export class PersonNameComponent implements OnInit {
 
-  constructor() { }
+  constructor(public sesionservice: SessionService) { }
```

Comment:
At the moment there are errors because the person-name component tries to access the name to display from the service before the round is started.

Reply:
Fixed with commit 46c0046